services/core-api/app/auth.py

```diff
@@ -36,18 +35,18 @@ def get_permission(self, mine_id: UUID):
 
 
 def get_mine_access():
+    from .api.users.minespace.models.minespace_user import MinespaceUser
```

Comment:
the imports from MinespaceUser before the app is initialized is what caused versioning to not work, so now they're loaded in the functions. There is a BE test added to catch the versioning breaking again.